### PR TITLE
Documentation for the new Recurring Actions feature

### PR DESCRIPTION
**Pull request by cbbayburt**
Adds documentation for the new "Recurring Actions" feature for 4.3.6, as an enhancement over the existing "Recurring Highstate" feature.

# Target branches

- [x] Master (to be released **after** 2023.04)
    Please wait for the code to be merged with https://github.com/uyuni-project/uyuni/pull/6678
- [x] Manager-4.3 (for 4.3.6)

# Links

Fixes https://github.com/SUSE/spacewalk/issues/21038

### CODE REVIEW

File: modules/administration/pages/actions.adoc [hunk 1]
```diff
@@ -16,14 +16,22 @@ Remote commands allows you to issue commands to individual Salt clients, or to a
 
 == Recurring Actions
 
-You can apply recurring actions on individual Salt clients, or to all clients in an organization.
+You can apply recurring actions on individual Salt clients, to a system group, or to all clients in an organization.
+
+Currently, {productname} supports the following action types as recurring actions:
+
+- [parameter]``Highstate:`` Execute the highstate
+- [parameter]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a configuration channel created by a user.
```
Comment (admd) [inline modules/administration/pages/actions.adoc:24]:
```suggestion
- [parameter]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a state channel created by a user.
```

nitpick ^^

File: modules/reference/pages/systems/system-details/sd-recurring-actions.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+The menu:Systems[Recurring Actions] section allows you to manage recurring actions on the selected client.
+You can schedule an arbitrary set of Salt states or the highstate to be executed at desired intervals.
+This is only available for Salt clients.
+
+For more information about Salt states, see xref:specialized-guides:salt/salt-states-and-pillars.adoc[].
+
+Currently, {productname} supports the following action types as recurring actions:
+
+- [guimenu]``Highstate:`` Execute the highstate
+- [guimenu]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a configuration channel created by a user.
```
Comment (admd) [inline modules/reference/pages/systems/system-details/sd-recurring-actions.adoc:13]:
```suggestion
- [guimenu]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a state channel created by a user.
```

**Review by admd — APPROVED** [review]
LGTM, thank you.

File: modules/administration/pages/actions.adoc (hunk 1 above)
Comment (cbbayburt) [reply modules/administration/pages/actions.adoc:24]: Actually, both state channels and normal config channels can be added as a state to recurring actions

File: modules/administration/pages/actions.adoc [hunk 3]
```diff
@@ -16,14 +16,22 @@ Remote commands allows you to issue commands to individual Salt clients, or to a
 
 == Recurring Actions
 
-You can apply recurring actions on individual Salt clients, or to all clients in an organization.
+You can apply recurring actions on individual Salt clients, to a system group, or to all clients in an organization.
+
+Currently, {productname} supports the following action types as recurring actions:
+
+- [parameter]``Highstate:`` Execute the highstate
```
Comment (keichwa) [inline modules/administration/pages/actions.adoc:23]:
```suggestion
- [parameter]``Highstate:`` Execute the highstate.
```

File: modules/reference/pages/systems/system-details/sd-recurring-actions.adoc [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+The menu:Systems[Recurring Actions] section allows you to manage recurring actions on the selected client.
+You can schedule an arbitrary set of Salt states or the highstate to be executed at desired intervals.
+This is only available for Salt clients.
+
+For more information about Salt states, see xref:specialized-guides:salt/salt-states-and-pillars.adoc[].
+
+Currently, {productname} supports the following action types as recurring actions:
+
+- [guimenu]``Highstate:`` Execute the highstate
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-recurring-actions.adoc:12]:
```suggestion
- [guimenu]``Highstate:`` Execute the highstate.
```

File: modules/reference/pages/systems/system-details/sd-recurring-actions.adoc [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- [guimenu]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a configuration channel created by a user.
+
+For more information about configuration channels, see xref:client-configuration:configuration-management.adoc[].
+
+
+
+== Creating Recurring Actions
+
+To create new schedules, click the btn:[Create] icon on the top right corner. Choose a supported action type, give your recurring schedule a name, select a time and day for it to recur, and click btn:[Create Schedule].
```
Comment (keichwa) [inline modules/reference/pages/systems/system-details/sd-recurring-actions.adoc:21]:
```suggestion
To create new schedules, click the btn:[Create] icon on the top right corner.
Choose a supported action type, give your recurring schedule a name, select a time and day for it to recur, and click btn:[Create Schedule].
```

**Review by keichwa — APPROVED** [review]
Thank you, Can!  Just 2 or 3 minor cosmetic observations.

**Comment by cbbayburt** [comment]
All done. Thanks for the reviews

**Review by jcayouette — APPROVED** [review]

**Pull request merged**